securedrop/journalist_templates/edit_account.html

```diff
@@ -27,14 +27,19 @@
 <p>{{ gettext('SecureDrop now uses automatically generated diceware passwords.') }}</p>
 <p>{{ gettext('Your password will be changed immediately, so you will need to save it before pressing the "Reset Password" button.') }}</p>
 
-{% if user %}
+{% if user and g.user != user %}
```

Comment:
Why is **g.user != user** necessary here ?  It's an extra security check ?

Reply:
It's basically to check if the admin user is updating their own password. I don't consider it so much a security check, but instead a protection from an admin changing their own password instead of another user and getting locked out.

Reply:
Good call ! 👍

Reply:
Nice. I'm pretty this actually resolves #2334.